src/main/java/domain/Cards.java

```diff
@@ -0,0 +1,92 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Cards {
+
+    private static final int BURST_SCORE = 22;
+
+    private final List<Card> cards;
+
+    public Cards() {
+        this.cards = new ArrayList<>();
+    }
+
+    public Cards(List<Card> cards) {
+        this.cards = cards;
+    }
+
+    public void add(Card card) {
+        cards.add(card);
+    }
+
+    public int calculateTotalPoint() {
+        int totalPoint = sumPointWithoutAce();
+        totalPoint = sumAcePoint(totalPoint);
+
+        return totalPoint;
+    }
+
+    private int sumPointWithoutAce() {
+        return cards.stream()
+                .filter(card -> !card.isAce())
+                .mapToInt(card -> card.getNumber().getPoint())
+                .sum();
+    }
+
+    private int sumAcePoint(int totalPoint) {
+        List<Number> aces = extractAces();
+
+        for (Number ace : aces) {
+            int acePoint = calculateAcePoint(totalPoint, ace);
+            totalPoint += acePoint;
```

Comment:
불필요한 변수를 제거할 수 있을듯하네요. 😃 

```suggestion
            totalPoint += calculateAcePoint(totalPoint, ace);
```

Reply:
적용 완료하였습니다!